test/In.ProjectEKA.HipServiceTest/DataFlow/DataFlowClientTest.cs

```diff
@@ -1,26 +1,31 @@
 namespace In.ProjectEKA.HipServiceTest.DataFlow
 {
     using System;
+    using System.Collections.Generic;
     using System.Net;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
-    using In.ProjectEKA.DefaultHip.DataFlow;
     using In.ProjectEKA.HipService.DataFlow;
-    using In.ProjectEKA.HipServiceTest.DataFlow.Builder;
+    using Builder;
+    using HipLibrary.Patient;
+    using HipLibrary.Patient.Model;
+    using Hl7.Fhir.Model;
+    using Microsoft.Extensions.DependencyInjection;
     using Moq;
     using Moq.Protected;
+    using Optional;
     using Xunit;
+    using Task = System.Threading.Tasks.Task;
 
     [Collection("Queue Listener Tests")]
     public class DataFlowClientTest
     {
-        private readonly Collect collect = new Collect("observation.json");
-
         [Fact]
-        private async Task ReturnSuccessOnDataFlow()
+        private async Task ShouldReturnDataComponent()
         {
             var handlerMock = new Mock<HttpMessageHandler>(MockBehavior.Strict);
+            var serviceScopeFactory = new Mock<IServiceScopeFactory>(MockBehavior.Strict);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'serviceScopeFactory'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5073610)